### PR TITLE
Add wordy to the prolog exercises

### PR DESCRIPTION
**Pull request by neenjaw**
This is a series of commits to add wordy to the prolog track.  I explored a few new topics to the track to see what interest there is in implementing them more broadly.

Summary of changes:
1. Added the [shebang](https://github.com/neenjaw/exercism-prolog/blob/a7741fc6102cb365d0a9b129403aaae033f7962b/bin/build.sh#L1) to the build shell script 
> defensive change from differing default shell
2. Added the '--build` argv flag to the build command
> This is because `wordy` is coded in the SWI-PL module style, when the test is loaded it imports the `wordy module` but since it is a stub, if loaded, an error occurs.  The `--build` argv flag prevents the module from being loaded.
3. added `wordy.example`, `wordy.pl`, and `wordy_tests.plt` for the exercise
4. added `.meta/version` for problem specification tracking
5. added `README.md` for the problem description
6. `config.json` updated and linted with `configlet`

### CODE REVIEW

**Comment by neenjaw** [comment]
As far as the new topics:

- introducing modularity (in the SWI-PL) style
- introducing throwing errors

**Comment by Average-user** [comment]
@neenjaw Thank you. I'm currently looking at it, but as I pointed out in #98 changes to `bin/_` should be done in separated PR. If your PR is named add `some-exercise` to prolog, you should only make commits that create and change files in `exercise/some-exercise`.

Other than that It seems great. I'll let you know if find anything else. Thank you and apologies for the time passed

**Comment by neenjaw** [comment]
@Average-user, no problem about time, I completely forgot about this PR. I can separate the PRs, as I don't even recall your other advice for the same. 

Thanks for reviewing. Let me know what else I can do before I make that revision.

**Comment by Average-user** [comment]
Great!, I don't think you should recall anything since it was done in a PR of other person :)

**Comment by Average-user** [comment]
By the way just in case, if you are interested in becoming maintainer of this track look at #97 . You would be more than welcomed

**Comment by neenjaw** [comment]
For sure, I will definitely consider it as prolog is very interesting to me. I am working with the elixir track as well on v3, and I think the v3 concept exercises would work really well for learning prolog

**Comment by Average-user** [comment]
@neenjaw Apart from moving the commit on `bin` to other PR, the only thing I would suggest is setting `difficulty` to 1. Of course this exercises is not of difficulty 1, but as for now all exercises are set to that. And a PR to fix all of them should be done separately. Or at least that was de conclusion at which we arrived in some conversation in slack that I currently can't find. But let me know what you think about that

**Comment by neenjaw** [comment]
That sounds fair! I'll try get these revisions written up and opened today or tomorrow.  Thanks for the review!

File: config.json [hunk 1]
```diff
@@ -159,6 +159,16 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": []
+    },
+    {
+      "slug": "wordy",
+      "uuid": "de3a48e8-6409-4a73-ac0f-66239b1d45c9",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 6,
```
Comment (neenjaw) [inline config.json:168]:
```suggestion
      "difficulty": 1,
```

File: exercises/wordy/wordy.example [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+math_expr(_, operator) -->
+  { throw(error(unknown_operation_error, operator_not_implemented)) }.
+
+operator("+") --> `plus`.
+operator("-") --> `minus`.
+operator("*") --> `multiplied by`.
+operator("/") --> `divided by`.
+operator(I)   -->
+  integer(I),
+  { throw(error(syntax_error, integer_found_rather_than_operator)) }.
```
Comment (neenjaw) [inline exercises/wordy/wordy.example:71]:
```suggestion
  { throw(error(syntax_error, integer_found_rather_than_operator)) }.

```

File: exercises/wordy/.meta/version [hunk 3]
```diff
@@ -0,0 +1 @@
+1.5.0
```
Comment (neenjaw) [inline exercises/wordy/.meta/version:1]:
```suggestion
1.5.0

```

**Comment by neenjaw** [comment]
This build will fail because fetch-configlet needs to be updated. See elixir's PR for more information here: https://github.com/exercism/elixir/pull/535

**Comment by neenjaw** [comment]
I have also removed the swi-pl module concept from this exercise so that the script at `bin/build.sh` does not need to be updated.

**Comment by Average-user** [comment]
Wouldn't it be smoother if we first merge the PR for `fetch-configlet`? I'm not sure, but I think tat would make the tests pass automatically. I remember doing it like this in exercisms haskell repo. But that was long ago.

**Comment by neenjaw** [comment]
Sounds good, see #99, I will rebase this before merge.

**Comment by Average-user** [comment]
@neenjaw Mhh apparently not working as I thought. I'm looking at if I can find a way to make travis rerun the tests

**Comment by Average-user** [comment]
@neenjaw Well thankyou now It seems we are good to go! I'll wait your confirmation before merge

**Comment by neenjaw** [comment]
I am good if you are! :)

**Pull request merged**